src/assets/scripts/main.js

```diff
@@ -1 +1,11 @@
 import 'ramda';
+var a = "1";
```

Comment:
__[ESLint]__ <sub>posted by Sider</sub>

Error - 'a' is assigned a value but never used. (no-unused-vars)